tests/test_flow/test_persistence_random.py

```diff
@@ -248,6 +281,8 @@ class RandomFlowTestConfig:
     n_iterations = attr.ib()
     p_persist = attr.ib(default=0.5)
     p_multi_out = attr.ib(default=0.3)
+    p_three_out_given_multi = attr.ib(default=0.3)
```

Comment:
Since you already have documentation for the older variables, it would be nice to add it for the new variables as well.

Reply:
Oh yeah! Thanks.